catalog-info.yaml

```diff
@@ -1,30 +1,25 @@
 # Backstage documentation
 # https://backstage.io/docs/features/software-catalog/descriptor-format/
 
-# !!! WARNING !!!
-# This is a template file with a number of fields that need to be filled before merging this to the default branch
 apiVersion: backstage.io/v1alpha1
 # Component, API, Template, Group, User, Resource, System, Domain, Location
-kind: unknown
+kind: API
```

Comment:
@roryscarson is this right? i haven't even been configuring this stuff on these client libraries and things that aren't services. @Sebb767 who is one of our contracted SDK maintainers took his best-effort attempts at filling this all out, but we could use your guidance :)

Reply:
The backstage docs above have descriptions for each one. I think Component or System would fit. I don't think the backstage validator checks the accuracy of this. For what it's worth almost everything we have currently is listed as a Component.

Reply:
k, perfect. @Sebb767 feel free to update your changes :)